src/server/scripts/Custom/Solocraft.cpp

```diff
@@ -284,6 +284,17 @@ class SolocraftAnnounce : public PlayerScript
             }
         }
     }
+
+    void OnLogout(Player* player) override
+    {
+        //Database query to see if an entry is still there
+        QueryResult result = CharacterDatabase.PQuery("SELECT `GUID` FROM `custom_solocraft_character_stats` WHERE GUID = %u", player->GetGUID());
```

Comment:
Do you need the select at all ? Isn't better to just send a DELETE  without select ?

Reply:
actually. u are right. It will be the same in the end. i will go ahead and make the amendment.